### PR TITLE
Local image store test

### PR DESCRIPTION
**Pull request by ilackarms**
based off #46

### CODE REVIEW

File: pkg/cmd/build/build.go [hunk 1]
```diff
@@ -89,7 +89,11 @@ func runBuild(opts buildOptions) error {
 			return err
 		}
 
-		tmpDir, err := ioutil.TempDir("/tmp", "wasme")
+		pwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		tmpDir, err := ioutil.TempDir(pwd, "wasme")
```
Comment (scootweiss) [inline pkg/cmd/build/build.go]: @yuval-k this was the change needed to fix CI; i believe we couldn't mount a directory outside of `/workspace` to the container in CI (though no error appears)

Comment (yuval-k) [reply pkg/cmd/build/build.go]: use TMPDIR var to fix this in CI

File: pkg/resolver/resolver.go [hunk 2]
```diff
@@ -45,7 +45,11 @@ func NewResolver(username, password string, insecure bool, plainHTTP bool, confi
 		}
 	}
 
-	token, _ := store.GetToken()
+	token, err := store.GetToken()
+	if err != nil {
+		// TODO: log err for pushes
+		//logrus.Warnf("Warning: No token found. Make sure to run `wasme login`: %v", err)
```
Comment (yuval-k) [inline pkg/resolver/resolver.go]: log debug

**Pull request merged**